docs/ChangeLog.md

```diff
@@ -1,7 +1,19 @@
 # Splunk Operator for Kubernetes Change Log
 
+## 2.2.0 (2023-01-24)
+
+* This is the 2.2.0 release. The Splunk Operator for Kubernetes is a supported platform for deploying Splunk Enterprise with the prerequisites and constraints laid out [here](https://github.com/splunk/splunk-operator/blob/main/docs/README.md#prerequisites-for-the-splunk-operator)
+
+* This release depends upon changes made concurrently in the Splunk Enterprise container images. You should use the splunk/splunk:9.0.3-a1 image with it
+
+* CSPL-1493 - Added support to automate the installation of the [Enterprise Security (ES) App](https://splunkbase.splunk.com/app/263) via appFramework phase 3
+
 ## 2.1.1 (2022-12-06)
 
+* This is the 2.1.1 release. The Splunk Operator for Kubernetes is a supported platform for deploying Splunk Enterprise with the prerequisites and constraints laid out [here](https://github.com/splunk/splunk-operator/blob/master/docs/README.md#prerequisites-for-the-splunk-operator)
+
+* This release depends upon changes made concurrently in the Splunk Enterprise container images. You should use the splunk/splunk:9.0.2 image with it
+
 * CSPL-2177: Fixed CVE-2022-42898 vulnerability
```

Comment:
We should remove this vulnerability name and just say fixed security vulnerability.